os/drivers/audio/Kconfig

```diff
@@ -0,0 +1,126 @@
+#
+# For a description of the syntax of this configuration file,
+# see the file kconfig-language.txt in the NuttX tools repository.
```

Comment:
NuttX, so let's change this line.
# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt